### ISSUE TITLE
M1 - mixed results

### ISSUE THREAD

**Issue by ofirgalcon** [tuxudo/memory#3]
On Macbook Air M1:

Memory and RAM pages are empty, so is memory.plist on client

On M1 MBP 13, 14 and 16

Memory page is OK. RAM page shows this: 
<img width="482" alt="image" src="https://user-images.githubusercontent.com/16665880/165177321-736e8b0b-0779-4c87-a8ce-979ecb574da4.png">

**Comment by ofirgalcon** [comment]
More info on this - here's what the various cmd in the script produce on MBA 13 M1. The MR plists in the cache are empty:

```system_profiler SPMemoryDataType
Memory:

      Memory: 8 GB
      Type: LPDDR4

vm_stat
Mach Virtual Memory Statistics: (page size of 16384 bytes)
Pages free:                                4578.
Pages active:                            125980.
Pages inactive:                          119294.
Pages speculative:                         6043.
Pages throttled:                              0.
Pages wired down:                         83039.
Pages purgeable:                            929.
"Translation faults":                 384755704.
Pages copy-on-write:                   24520130.
Pages zero filled:                    161400735.
Pages reactivated:                     29338381.
Pages purged:                           2509474.
File-backed pages:                        97793.
Anonymous pages:                         153524.
Pages stored in compressor:              409720.
Pages occupied by compressor:            150760.
Decompressions:                        36651147.
Compressions:                          41608726.
Pageins:                               38031389.
Pageouts:                                323435.
Swapins:                               10301312.
Swapouts:                              11100180.


memory_pressure
The system has 8589934592 (524288 pages with a page size of 16384).

Stats:
Pages free: 3751
Pages purgeable: 559
Pages purged: 2509902

Swap I/O:
Swapins: 10301348
Swapouts: 11100180

Page Q counts:
Pages active: 126248
Pages inactive: 120223
Pages speculative: 5620
Pages throttled: 0
Pages wired down: 83286

Compressor Stats:
Pages used by compressor: 150628
Pages decompressed: 36651529
Pages compressed: 41608726

File I/O:
Pageins: 38031598
Pageouts: 323435

System-wide memory free percentage: 52%


sysctl vm.swapusage
vm.swapusage: total = 2048.00M  used = 579.38M  free = 1468.62M  (encrypted)```

**Comment by ofirgalcon** [comment]
More - here's the plist output of `system_profiler SPMemoryDataType -xml`

```<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<array>
	<dict>
		<key>_SPCommandLineArguments</key>
		<array>
			<string>/usr/sbin/system_profiler</string>
			<string>-nospawn</string>
			<string>-xml</string>
			<string>SPMemoryDataType</string>
			<string>-detailLevel</string>
			<string>full</string>
		</array>
		<key>_SPCompletionInterval</key>
		<real>0.045794010162353516</real>
		<key>_SPResponseTime</key>
		<real>0.06650996208190918</real>
		<key>_dataType</key>
		<string>SPMemoryDataType</string>
		<key>_detailLevel</key>
		<integer>-1</integer>
		<key>_items</key>
		<array>
			<dict>
				<key>SPMemoryDataType</key>
				<string>8 GB</string>
				<key>dimm_type</key>
				<string>LPDDR4</string>
			</dict>
		</array>
		<key>_parentDataType</key>
		<string>SPHardwareDataType</string>
		<key>_properties</key>
		<dict>
			<key>SPMemoryDataType</key>
			<dict>
				<key>_isColumn</key>
				<string>YES</string>
				<key>_order</key>
				<string>12</string>
			</dict>
			<key>_name</key>
			<dict>
				<key>_isColumn</key>
				<string>YES</string>
				<key>_isOutlineColumn</key>
				<string>YES</string>
				<key>_order</key>
				<string>10</string>
			</dict>
			<key>dimm_ecc_errors</key>
			<dict>
				<key>_order</key>
				<string>60</string>
			</dict>
			<key>dimm_serial_number</key>
			<dict>
				<key>_detailLevel</key>
				<string>0</string>
			</dict>
			<key>dimm_size</key>
			<dict>
				<key>_isColumn</key>
				<string>YES</string>
				<key>_order</key>
				<string>20</string>
			</dict>
			<key>dimm_speed</key>
			<dict>
				<key>_isColumn</key>
				<string>YES</string>
				<key>_order</key>
				<string>40</string>
			</dict>
			<key>dimm_status</key>
			<dict>
				<key>_isColumn</key>
				<string>YES</string>
				<key>_order</key>
				<string>50</string>
			</dict>
			<key>dimm_type</key>
			<dict>
				<key>_isColumn</key>
				<string>YES</string>
				<key>_order</key>
				<string>30</string>
			</dict>
			<key>volumes</key>
			<dict>
				<key>_detailLevel</key>
				<string>0</string>
			</dict>
		</dict>
		<key>_timeStamp</key>
		<date>2022-05-26T17:33:39Z</date>
		<key>_versionInfo</key>
		<dict>
			<key>com.apple.SystemProfiler.SPMemoryReporter</key>
			<string>80</string>
		</dict>
	</dict>
</array>
</plist>```

**Comment by tuxudo** [comment]
Issues resolved with version v2.1 of module.
